app/cors/server/cors.js

```diff
@@ -48,6 +48,9 @@ WebApp.rawConnectHandlers.use(function(req, res, next) {
 	if (/^\/(api|_timesync|sockjs|tap-i18n)(\/|$)/.test(req.url)) {
 		res.setHeader('Access-Control-Allow-Origin', '*');
 	}
+	if (!settings.get('Allow_Loading_In_Iframe')) {
```

Comment:
This is a bit confusing, I'd suggest to change `Allow_Loading_In_Iframe` to `Iframe_Restrict_Access`, and when enabled (the default behavior) set the `X-Frame-Options`. In this case I would change the setting `X_Frame_Options` to `Iframe_X_Frame_Options `